components/mosquitto/examples/broker/README.md

```diff
@@ -13,6 +13,9 @@ If you enabled also the mqtt client, this example will connect to the local brok
 
 You can connect to the ESP32 mosquitto broker using some other client using the ESP32 IPv4 address and the port specified in the project configuration menu.
 
+> [!IMPORTANT]
+> Please do not reuse the test certificates and keys used in this example. Note that these are single purpose, self-signed with common name set to `"127.0.0.1"`.
```

Comment:
The test certificates and key files should be left blank. It would also be helpful to add a section explaining how to get the certificates and keys.

Reply:
rephrase to stress out that these certs are used for demo purpose (but keep the certs).

maybe reference to https server about the instructions on how to get certs/keys

Reply:
updated